python/paddle/distributed/passes/cpp_pass.py

```diff
@@ -26,3 +26,16 @@ def cpp_name(self):
 
     def _type(self):
         return PassType.FUSION_OPT
+
+
+@register_pass("fuse_relu_depthwise_conv_pass")
```

Comment:
I prefer to use the name `fuse_relu_depthwise_conv`, which is of the same style with `fuse_elewise_add_act`.